iianmelo/citigames-squadian
# Feat/page/creategame

What I did
- created the creategame folder in components
- added the page ceategame
- created the colors and others variable for the creategame page (style/global and tailwind.config)

How to test
	1	Open the terminal and go to the folder of this project
	2	In terminal:
	i	git checkout main
	ii	git pull
	iii	git checkout <feat/component/creategame>
	iv	git pull origin <feat/component/creategame>
	v	yarn install
	vii	yarn dev
	3	In your browser go to http://localhost:3000
	4	Check if the component looks just like the mockup in different screen sizes
	5	In VScode open the relative path of the test page
	6	Test the components/pages that were made
	7	Read the changed files section on GitHub
	8	Test the things that you think are worth testing, even the ones that are not in this description
